2017-08-30 02:03:03: -- Analysis Started by On File Change Incremental
2017-08-30 02:03:03: 变更：9， 冲突：， 复制时间：， 复制状态：， 错误： , All: 239
2017-08-30 02:03:03: Left to Right: Copy File: 1 Rename/Move File/Folder: 8 
2017-08-30 02:03:03: == Sync Started by On File Change Incremental
2017-08-30 02:03:04: 移动文件 mtp://S60_Handset/Memory card (ESD-ISO)/Python/env/PIPS.SIS -> mtp://S60_Handset/Memory card (ESD-ISO)/Python/env/s60v3/PIPS.SIS
2017-08-30 02:03:06: 移动文件 mtp://S60_Handset/Memory card (ESD-ISO)/Python/env/pips.sisx -> mtp://S60_Handset/Memory card (ESD-ISO)/Python/env/s60v3/pips.sisx
2017-08-30 02:03:06: 移动文件 mtp://S60_Handset/Memory card (ESD-ISO)/Python/env/pips_compact.sis -> mtp://S60_Handset/Memory card (ESD-ISO)/Python/env/s60v3/pips_compact.sis
2017-08-30 02:03:08: 移动文件 mtp://S60_Handset/Memory card (ESD-ISO)/Python/env/Python_2.0.0.sis -> mtp://S60_Handset/Memory card (ESD-ISO)/Python/env/s60v3/Python_2.0.0.sis
2017-08-30 02:03:08: 新建复制 'D:/Projects/SymbianPython/python_working/env/s60v3/新建文本文档.txt' -> 'mtp://S60_Handset/Memory card (ESD-ISO)/Python/env/s60v3/新建文本文档.txt'
- 错误: Copying file: Cannot open local file 'D:/Projects/SymbianPython/python_working/env/s60v3/新建文本文档.txt' for read: 系统找不到指定的文件。  (error 2)
2017-08-30 02:03:08: ===== ：同步完毕。时间：00:00:05，速度： 819 字节/s =====, 项目同步未必： 8，错误： 1
2017-08-30 02:03:08: 
